mteb/tasks/Classification/eng/ArxivClassification.py

```diff
@@ -0,0 +1,43 @@
+from __future__ import annotations
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks import AbsTaskClassification
+
+
+class ArxivClassification(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="ArxivClassification",
+        description="Classification Dataset of Arxiv Papers",
+        dataset={
+            "path": "ccdv/arxiv-classification",
+            "revision": "f9bd92144ed76200d6eb3ce73a8bd4eba9ffdc85",
+        },
+        reference="https://ieeexplore.ieee.org/document/8675939",
+        type="Classification",
+        category="s2s",
+        eval_splits=["test"],
+        eval_langs=["eng-Latn"],
+        main_score="accuracy",
+        date=None,
```

Comment:
A rough estimate will do: (yyyy-mm-dd, yyyy-mm-dd)

Reply:
Is this the date this dataset was compiled or the dates of the papers included in this dataset?

Reply:
Compiled. But what you have is fine.